io.github.JakubMelka.Pdf4qt.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/Pdf4QtViewerProfi",
+    "build-options": {
+	"build-args": [ "--share=network" ]
+	},
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc",
+        "--filesystem=xdg-config/kdeglobals:ro"
```

Comment:
why do you need access to this?

Reply:
I have deleted "--filesystem=xdg-config/kdeglobals:ro" - I have heard that some settings is here for Qt applications, but it is probably not needed.

Reply:
Other feature are needed - my software can render PDF pages on GPU.